Sources/FluidAudio/Diarizer/Core/DiarizerManager.swift

```diff
@@ -342,6 +346,47 @@ public final class DiarizerManager {
         return (segments, timings)
     }
 
+    private func extractEmbeddingsSynchronously<C>(
```

Comment:
a wrapper that is needed because we don't want to surface teh async/await to the end user.